webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml

```diff
@@ -929,4 +933,31 @@
     </component-plugin>
   </external-component-plugins>
 
+  <external-component-plugins>
+    <target-component>org.exoplatform.services.jcr.RepositoryService</target-component>
+    <component-plugin>
+      <name>add.namespaces</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.services.jcr.impl.AddNamespacesPlugin</type>
+      <init-params>
+        <properties-param>
+          <name>namespaces</name>
+          <property name="oo" value="http://www.exoplatform.com/jcr/officeonline/1.0/" />

```

Comment:
`oo` doesn't tell clearly what is it, let's use more obvious name `msoffice`

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/18/commits/34bebb38165ffcb6c2e89d0bf3a98b8d041d7593